README.md

```diff
@@ -7,17 +7,20 @@
 - If the program crashes, create an issue with the error.
 
 ## INSTALLATION:
-1. Install Python 3.10.6 from https://www.python.org/downloads/windows/
-   - Note that Python 3.10.6 cannot be used on Windows 7 or earlier.
+1. Install Python 3.11.4 from https://www.python.org/downloads/windows/
+   - Note that Python 3.11.4 cannot be used on Windows 7 or earlier.
 2. Clone the repository or download it from here https://github.com/jfd02/TFT-OCR-BOT/archive/refs/heads/main.zip
 3. Open Command Prompt and change the current directory to the folder where main.py is located 
-4. Run pip install -r requirements.txt in Command Prompt
-5. Install tesseract using the Windows installer available at: https://github.com/UB-Mannheim/tesseract/wiki
+4. Run 'pip install -r requirements.txt' in Command Prompt
+5. Install tesseract 5.3.1.20230401 using the Windows installer available at: https://github.com/UB-Mannheim/tesseract/wiki
    - Note the tesseract path from the installation.
-   - Set the tesseract path in the settings.py file (it may already be correct)
-6. Configure settings.py so the league client path is correct
-7. Disable all in-game overlays
-8. Run the main.py file
+   - Set Tesseract tessdata folder path in settings.py file (probably already correct)
+6. Download tesserocr v2.6.0 at: https://github.com/simonflueckiger/tesserocr-windows_build/releases
+7. Open Command Prompt and change the current directory to the folder where tesserocr-2.6.0-cp311-cp311-win_amd64.whl is located
+8. Run 'pip install tesserocr-2.6.0-cp311-cp311-win_amd64.whl' in Command Prompt
```

Comment:
Is there a way to simplify these steps? I think a lot of people will get stuck here and end up opening a lot of issues. I think the performance improvement is worth it anyways but just wondering if we can turn this into some sort of install script or something similar

Reply:
I downloaded new files and followed steps everything works fine

Reply:
I have just wrote a bat script to install requirements.txt and tesserocr.
All we need to do is download tesserocr installation file into the bot's folder and run the bat script.

Reply:
For those who are already using this bot and using Python 3.10, it is not mandatory to update to 3.11 because its installation is more complex. However, installing the new version of Tesseract will automatically uninstall the old version, which is relatively simple. Therefore, I have specified the Tesseract and its corresponding TesserOCR version in the readme.md file.

Reply:
about moving from python 3.10 to 3.11 i moved to 3.11 by remove installed packages through pip list and uninstalled python itself then installed 3.11 and new requirments.txt and so on 

is that what should be or Iam missing something.

Reply:
> Also i wnna ask if i can run bot in vmware workstation, any idea to run bot while using pc? i'm sure u tried to do that before :D

**Dont do that.** RIOT will ban any account played on virtual machine.

Reply:
ty for these information.